app/src/main/res/layout/fragment_object_set.xml

```diff
@@ -148,4 +148,12 @@
         android:visibility="gone"
         tools:visibility="visible" />
 
+    <androidx.compose.ui.platform.ComposeView
+        android:id="@+id/composeView"
```

Comment:
consider renaming this to something more concrete, maybe `templatePickerView`?

Reply:
renamed to `templatesWidget`